app/styles/index.js

```diff
@@ -1,4 +1,8 @@
 import Theme from './theme.js'
+import devices from './devices.js'
 import './global_styles.js'
 
-export default Theme
\ No newline at end of file
+export {
+    Theme,
```

Comment:
its just the default Theme and its constant. I wanted to differentiate it from the redux store theme that is lowercase